### PR TITLE
megatools 1.9.97

### PR DESCRIPTION
**Pull request by G-Square**

### CODE REVIEW

File: Library/Formula/megatools.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class Megatools < Formula
   desc "Command-line client for Mega.co.nz"
   homepage "https://megatools.megous.com/"
-  url "https://megatools.megous.com/builds/megatools-1.9.96.tar.gz"
-  sha256 "c657a0988223aada021a7a6ed408227fc66435de7248f23952ad7adf9e1242f0"
+  url "https://megatools.megous.com/builds/megatools-1.9.97.tar.gz"
+  sha256 "3316c56ee55adef0ab113aad41ef95eb28fc15218639f69b1e04362b1c725af3"
```
Comment (sjackman) [inline Library/Formula/megatools.rb]: Please remove this chunk. New versions are added upstream by `Homebrew/homebrew`.

File: Library/Formula/megatools.rb [hunk 2]
```diff
@@ -27,7 +28,7 @@ def install
   test do
     # Downloads a publicly hosted file and verifies its contents.
     system "#{bin}/megadl",
-      "https://mega.co.nz/#!3Q5CnDCb!PivMgZPyf6aFnCxJhgFLX1h9uUTy9ehoGrEcAkGZSaI",
+      "https://mega.nz/#!3Q5CnDCb!PivMgZPyf6aFnCxJhgFLX1h9uUTy9ehoGrEcAkGZSaI",
```
Comment (sjackman) [inline Library/Formula/megatools.rb]: Is this chunk necessary? It looks to be redirected automatically. Feel free to send this chunk upstream to `Homebrew/homebrew` and remove it here.

File: Library/Formula/megatools.rb [hunk 3]
```diff
@@ -15,6 +15,7 @@ class Megatools < Formula
   depends_on "glib"
   depends_on "glib-networking"
   depends_on "openssl"
+  depends_on "XML::Parser" => :perl unless OS.mac?
```
Comment (sjackman) [inline Library/Formula/megatools.rb]: This chunk is good. Thanks!

File: Library/Formula/megatools.rb (hunk 2 above)
Comment (G-Square) [inline Library/Formula/megatools.rb]: necessary? No, but does it save some time (ms-2sec) on redirecting? yes

**Comment by sjackman** [comment]
Please keep the `depends_on` chunk and remove the other two chunks.
Please squash all the commits to a single commit.
```sh
git rebase -i origin/master
# select squash
git push G-Square master --force
```

File: Library/Formula/megatools.rb (hunk 1 above)
Comment (G-Square) [inline Library/Formula/megatools.rb]: So, you don't want the new version?

File: Library/Formula/megatools.rb (hunk 2 above)
Comment (sjackman) [inline Library/Formula/megatools.rb]: Please submit this chunk upstream to `Homebrew/homebrew`. See [Contributing to Linuxbrew](https://github.com/Linuxbrew/linuxbrew/blob/master/.github/CONTRIBUTING.md#contributing-to-linuxbrew)

> Patches to fix issues that you have reproduced on both Linuxbrew and Homebrew should be sent to Homebrew.

**Comment by sjackman** [comment]
Did you intend to close this pull request?

**Pull request closed without merging**